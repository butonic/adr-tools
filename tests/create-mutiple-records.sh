adr new First Decision
adr new Second Decision
adr new Third Decision
ls doc/adr
head -10 doc/adr/0002-second-decision.md
head -10 doc/adr/0003-third-decision.md
