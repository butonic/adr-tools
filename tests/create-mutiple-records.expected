adr new First Decision
doc/adr/0001-first-decision.md
adr new Second Decision
doc/adr/0002-second-decision.md
adr new Third Decision
doc/adr/0003-third-decision.md
ls doc/adr
0001-first-decision.md
0002-second-decision.md
0003-third-decision.md
head -10 doc/adr/0002-second-decision.md
---
title: "2. Second Decision"
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/ocis/adr
geekdocFilePath: 0002-second-decision.md
---

* Status: proposed
* Deciders: [list everyone involved in the decision] <!-- optional -->
* Date: 1992-01-12
head -10 doc/adr/0003-third-decision.md
---
title: "3. Third Decision"
geekdocRepo: https://github.com/owncloud/ocis
geekdocEditPath: edit/master/docs/ocis/adr
geekdocFilePath: 0003-third-decision.md
---

* Status: proposed
* Deciders: [list everyone involved in the decision] <!-- optional -->
* Date: 1992-01-12
